qutebrowser/browser/webengine/webview.py

```diff
@@ -258,13 +259,51 @@ def acceptNavigationRequest(self,
         self.navigation_request.emit(navigation)
         return navigation.accepted
 
+    @staticmethod
```

Comment:
We don't really use staticmethods much. I wonder if this should just be a regular function in this module? No hard feelings either way though.